src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Scanner;
+
 public class Duke {
+    static final String LOGO = "   _____                .__   \n" +
+            "  /  _  \\ ___  ___ ____ |  |  \n" +
+            " /  /_\\  \\\\  \\/  // __ \\|  |  \n" +
+            "/    |    \\>    <\\  ___/|  |__\n" +
+            "\\____|__  /__/\\_ \\\\___  >____/\n";
+    public static void greetUser(){
+        String greeting = "____________________________________________________________\n"
+        + "Hello! My name is Axel. :-)\n"
+        + "How may I help you today?\n"
+        + "____________________________________________________________";
+        System.out.println(greeting);
+    }
+
+    public static void sayByeToUser(){
+        String message = "____________________________________________________________\n"
+        + "Goodbye. Hope to see you again soon!\n"
+        + "____________________________________________________________";
+        System.out.println(message);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(LOGO);
+        greetUser();
+        Scanner input = new Scanner(System.in);
+        TaskManager TaskManager1 = new TaskManager();
+        boolean isFinished = false; //variable to indicate if the program should be terminated
```

Comment:
perhaps programIsFinished?
once again, isFinished doesn't really mean anything on its own, if it suddenly appears somewhere far down in the code